### PR TITLE
[T-6315][15.0][MIG] ecommerce_connector: migration to 15.0

### PR DESCRIPTION
**Pull request by manuelregidor**
@ValentinVinagre @HaraldPanten  @luis-ron 

@Tisho99 Could you review, please?

T-6315

### CODE REVIEW

**Review by Tisho99 — COMMENTED** [review]
If we are un version 15, Why there are no tests?

If you migrate all the commit history, Why there is no pre-commit auto fixes commit?

**Comment by Tisho99** [comment]
@manuelregidor Can you do the pre-commit auto fixes commit?

Reviewing the changes in the logic with the pre-commit changes is a lot harder

**Comment by manuelregidor** [comment]
@Tisho99 Pre-commit commit added. We are not going to include tests in this module for now.

File: ecommerce_connector/models/ecommerce_connector.py [hunk 1]
```diff
@@ -1140,8 +1258,10 @@ def _check_shipping_lines(self, values, move, errors):
                 ):
                     errors = self._write_errors(
                         errors,
-                        "%s in sales currency in shipping line with ID %s does not match the value sent with the call."
-                        % (compare_field, line.get("id")),
+                        "%s in sales currency in shipping line with ID {} does "
+                        "not match the value sent with the call.".format(
+                            compare_field,
+                        ),
```
Comment (Tisho99) [inline ecommerce_connector/models/ecommerce_connector.py]: You forgot one format parameter (line.get("id"))

                        "{} in sales currency in shipping line with ID {} does "
                        "not match the value sent with the call.".format(
                            compare_field,
                            line.get("id")
                        ),

File: ecommerce_connector/models/ecommerce_connector.py [hunk 2]
```diff
@@ -1252,23 +1369,154 @@ def _check_credit_lines(self, errors, invoice_id, lines, ecommerce_connection):
...
+                    lambda a, product=product_id: a.product_id == product
                 ).mapped("quantity")
             )
             if float(line.get("quantity")) > product_lines_qty:
                 errors = self._write_errors(
                     errors,
-                    "Quantity of product with Ecommerce ID %s is higher than the quantity of product in the invoice."
-                    % line.get("productId"),
+                    "Quantity of product with Ecommerce ID %s is higher than "
+                    "the quantity of product in the invoice.".format(),
```
Comment (Tisho99) [inline ecommerce_connector/models/ecommerce_connector.py]: You also skipped a format parameter here:

                    "Quantity of product with Ecommerce ID {} is higher than "
                    "the quantity of product in the invoice.".format(line.get("productId")),

**Review by Tisho99 — APPROVED** [review]
LGTM, minor fixes

**Comment by manuelregidor** [comment]
@Tisho99 Changes applied

**Pull request merged**